.github/workflows/deploy-to-paas.yml

```diff
@@ -4,91 +4,58 @@ on:
   workflow_call:
     inputs:
       environment: { required: true, type: string }
-      cf_space_name: { required: true, type: string }
-      app_name: { required: false, type: string }
+      cf-space-name: { required: true, type: string }
+      app-name-prefix: { required: false, type: string }
       url: { required: false, type: string }
       instances: { required: false, type: number, default: 1 }
-      google_tag_id: { required: false, type: string, default: GTM-PFTQ6G2 }
-      rolling_deployment: { required: false, type: boolean, default: false }
-      use_stub_zendesk: { required: false, type: boolean, default: true }
+      rolling-deployment: { required: false, type: boolean, default: false }
+      google-tag-id: { required: false, type: string, default: GTM-PFTQ6G2 }
+      use-stub-zendesk: { required: false, type: boolean, default: true }
```

Comment:
Does this create a danger that we one day accidentally deploy to prod with the stub Zendesk?

Reply:
On the flip side we'd risk deploying to non-prod with _real_ Zendesk. the default hasn't changed in this PR. I don't think it matters, we neeed to ensure we deploy to our envs with the correct config either way.